src/AlgebraicPetri.jl

```diff
@@ -121,19 +109,18 @@ end
         # reimplementation of pushout of Span{FinSetFunc, FinSetFun}
```

Comment:
This shouldn't be necessary anymore since calling `pushout` on FinFunctions now returns a [`CompositePushout`](https://github.com/AlgebraicJulia/Catlab.jl/blob/414b9787f90014c33a7b125d50554ebc044d4513/src/categorical_algebra/Limits.jl#L280). You can fish the coequalizer out of it if you have to.

Reply:
In principle, we shouldn't have to do *that* either because it breaks the abstraction boundary for the pushout. See the docs for [`CompositePullback`](https://github.com/AlgebraicJulia/Catlab.jl/blob/414b9787f90014c33a7b125d50554ebc044d4513/src/categorical_algebra/Limits.jl#L249). I was going to say that you should just use the universal property of the pushout, which is now implemented, but reminding myself about decorated cospans I see that it actually uses properties of the both coproduct *and* pushout. So I see two possibilities here:

1. Keep what you currently have because you actually need both the coproduct and coequalizer/pushout.
2. Compute the coproduct *and* the pushout; take the legs of the pushout, which are morphisms into the pushout object; then apply the universal property of the coproduct to these morphisms, giving a morphism from the coproduct object to the pushout object.

Reply:
Anyway, I figure this will get resolved anyway when we have a generic implementation of decorated/structured cospans for C-sets, so I wouldn't worry about it too much.

Reply:
Yeah, I think it would be ok to leave it as is until the generic structured cospans is implemented for 𝒞-sets